src/ComScoreStreamingAnalytics.ts

```diff
@@ -54,11 +59,24 @@ export class ComScoreStreamingAnalytics {
    * another
    * @param metadata
    */
-
   updateMetadata(metadata: ComScoreMetadata): void {
     this.metadata = metadata;
   }
 
+  /**
+   * sets the userContent to granted. Use after the ComScoreAnalytics object has been started
+   */
+  public userConsentGranted() {
```

Comment:
Why this name? Why not creating a setter like `setUserConsent(value: ComScoreUserConsent)` or `setUserConsentGranted`. 

This method name does not really tell what it does IMO.

Reply:
Rather than setting a value (which is not stored or retreived) we thought it made more sense to have these methods that they can call when the user changes their preference

Reply:
Ok I guess it's fine then. I personally would anyway go with the `set` prefix but I leave it up to you.